modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -104,6 +115,18 @@ struct GAPI_Streaming: public ::testing::TestWithParam<KernelPackage> {
         }
         throw std::logic_error("Unknown package");
     }
+
+    cv::GCompileArgs getCompileArgs() {
+        using namespace cv::gapi;
+        auto args = cv::compile_args(use_only{pkg});
+        if (cap) {
```

Comment:
why `cap` should be initialized at this point, how do you guarantee this?

Reply:
There is initialization in `GAPI_Streaming` ctor